test/Writer/Renderer/Extension/PodcastIndex/FeedTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasTest\Feed\Writer\Renderer\Extension\PodcastIndex;
+
+use Laminas\Feed\Writer;
+use Laminas\Feed\Writer\Renderer;
+use PHPUnit\Framework\TestCase;
+
+class FeedTest extends TestCase
+{
+    protected Writer\Feed $validWriter;
+
+    protected function setUp(): void
+    {
+        Writer\Writer::reset();
+
+        $this->validWriter = new Writer\Feed();
+        $this->validWriter->setTitle('This is a test feed.');
+        $this->validWriter->setDescription('This is a test description.');
+        $this->validWriter->setLink('http://www.example.com');
+        $this->validWriter->setType('rss');
+    }
+
+    protected function tearDown(): void
+    {
+        Writer\Writer::reset();
+    }
+
+    public function testRendersLockedTag(): void
+    {
+        $locked = [
+            'value' => 'yes',
+            'owner' => 'john.doe@example.com',
+        ];
+        $this->validWriter->setPodcastIndexLocked($locked);
+
+        $rssFeed = new Renderer\Feed\Rss($this->validWriter);
+        /** @var string $xml */
```

Comment:
If you're worried this might not be a string, add an assertion after the assignment, otherwise just drop it to stop Psalm complaining.